tests/common/fixtures/elasticsearch.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 
 import os
+import elasticsearch_dsl
```

Comment:
I believe following our import ordering conventions this should be grouped with the `pytest` import below as a third-party package.